translate/package.json

```diff
@@ -46,6 +46,7 @@
     "build": "rollup -c",
     "build:prod": "rollup -c --environment BUILD:production",
     "test": "jest",
+    "vitest": "vitest run",
```

Comment:
If this command is temporary and isn't run by CI, it should be left out -- it's just as easy to type
```
npx vitest run
```
as it is to type
```
npm run vitest
```